core/field_image.js

```diff
@@ -70,13 +71,32 @@ Blockly.FieldImage = function(src, width, height,
   this.size_ = new Blockly.utils.Size(imageWidth,
       imageHeight + Blockly.FieldImage.Y_PADDING);
 
-  this.flipRtl_ = opt_flipRtl;
-  this.text_ = opt_alt || '';
-  this.setValue(src || '');
-
+  // Deal with config options and param reordering.
```

Comment:
Since all this functionality is related, maybe we can move it to its own private method. something like ``configure_(options)``

Reply:
I changed it, but it feels like the backwards compatibility code makes less sense when it's extracted from its context.